src/project.rs

```diff
@@ -333,6 +333,25 @@ impl FuzzProject {
         Ok(debug)
     }
 
+    /// Prints the debug output of an input test case
+    pub fn debug_fmt_input(&self, debugfmt: &options::Fmt) -> Result<()> {
+        if !debugfmt.input.exists() {
+            bail!("Input test case does not exist.");
+        }
+
+        if let Ok(debug) =
+            self.run_fuzz_target_debug_formatter(&debugfmt.build, &debugfmt.target, &debugfmt.input)
+        {
+            eprintln!("Output of `std::fmt::Debug`:\n");
+            for l in debug.lines() {
+                eprintln!("{}", l);
+            }
+        } else {
+            bail!("Unable to return debug-formatted output for input test case.")
```

Comment:
Instead of doing `if let Ok(..) = { .. } else { .. }` and creating a new error in the not-`Ok` case, let's use [the `with_context` method from `anyhow`](https://docs.rs/anyhow/1.0.27/anyhow/trait.Context.html#tymethod.with_context) to add extra info to the error returned from `run_fuzz_target_debug_formatter`:

```rust
let debug = self.run_fuzz_target_debug_formatter(&debugfmt.build, &debugfmt.target, &debugfmt.input)
    .with_context(|| format!("failed to run `cargo fuzz fmt` on input:", debugfmt.input.display()))?;
```

This should be both more concise and provide more information when errors occur.